.storybook/_container.scss

```diff
@@ -1,3 +1,6 @@
+@import '~carbon-components/scss/globals/scss/feature-flags';
+
+$feature-flags: map-merge($default-feature-flags,  (ui-shell: true));
```

Comment:
I believe this line will be removed before the code is merged? (BTW [`export CARBON_USE_EXPERIMENTAL_FEATURES=true` will let you see the UI shell without changing code](https://github.com/IBM/carbon-components-react/blob/v6.40.0/.storybook/webpack.config.js#L69))

Reply:
Could we leave it so that the built storybook will have the UI Shell rendering correctly? Otherwise I can update the build process, I think. Just wanted to practice using the feature flags the way other teams would to be honest 👍